spec/app/serializers/unitary_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class UnitarySerializer < ActiveModel::Serializer
+  attributes :foo, :extra
+
+  def extra
+    'extra'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.